bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractZkLedgerManager.java

```diff
@@ -144,7 +143,7 @@ public void run() {
                     }
                 }
             } else {
-                LOG.warn("Failed on read ledger metadata of ledger {} : {}", ledgerId, rc);
+                LOG.warn("Failed on read ledger metadata of ledger {}", ledgerId, exception);
```

Comment:
I don't think the exception stack trace is useful here (in async callbacks). so I think we should still use `rc` here.